<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GCNewElementForm</class>
 <widget class="QWidget" name="GCNewElementForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Element</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/gcresources.qrc">
    <normaloff>:/resources/goblinicon.jpg</normaloff>:/resources/goblinicon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
  background-color: rgba(120, 120, 120, 255);
  font: 20px;
}

QDialog
{
  background-color: rgba(150, 150, 150, 255);
}

QMessageBox
{
  background-color: rgba(120, 120, 120, 255);
}

QPushButton
{
  color: rgb(0, 0, 0);
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 240), stop:1 rgba(150, 150, 160, 255));
  border-width: 1px;
  border-radius: 5px;
  border-color: gray;
  border-style: outset;
  font: 12px;
  min-width: 5em;
  min-height: 1em;
  padding: 3px;
}

QPushButton:pressed
{
  color: rgb(190, 190, 190);
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(150, 150, 160, 240), stop:1 rgba(110, 110, 125, 255));
  border-style: inset;
}

QPushButton:default 
{
   border-color: gray;
   border-width: 2px;   
   background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(190, 190, 190, 255));
 }

QToolButton
{
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(150, 150, 160, 255));
  border-width: 1px;
  border-radius: 5px;
  border-color: rgb(40, 40, 40);
  border-style: outset;
}

QToolButton:pressed
{
  background-color: qlineargradient(spread:reflect, x1:0.5, y1:0.5, x2:0.5, y2:1, stop:0 rgba(110, 110, 110, 255), stop:1 rgba(150, 150, 160, 255));
  border-style: inset;
}

QLabel
{
  background-color: rgba(140, 140, 140, 200);
  border-radius: 6px;
  padding: 1px;
  font: 12px;
}

QLineEdit
{
   border-width: 1px;
   border-radius: 5px;
   border-color: rgb(90, 90, 90);
   border-style: solid;
   padding: 2 4px;
   background-color: rgb(180, 180, 180);
   selection-background-color: rgb(98, 96, 122);
   font: 12px;
 }

QPlainTextEdit
{
   background-color: rgb(200,200,200);
   selection-background-color: rgba(120, 120, 130, 200);
   font: 12px;
}

QScrollBar:vertical 
{
   width: 17px;
   margin: 18px 0 18px 0;
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(140, 140, 140, 200), stop:1 rgba(160, 160, 160, 255));
   border-width: 1px;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::handle:vertical 
{
   background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(150, 150, 150, 255));
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/scrollgrip.png&quot;);
   min-height: 3em;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::add-line:vertical 
{
   border-color: rgba(40, 40, 40, 0);
   height: 18px;
   subcontrol-position: bottom;
   subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical 
{
   border-color: rgba(40, 40, 40, 0);
   height: 18px;
   subcontrol-position: top;
   subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/uparrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::up-arrow:vertical:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::down-arrow:vertical 
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/downarrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::down-arrow:vertical:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{
   background: none;
}

QScrollBar:horizontal 
{
   height: 17px;
   margin: 0 18px 0 18px;
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(140, 140, 140, 200), stop:1 rgba(160, 160, 160, 255));
   border-width: 1px;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::handle:horizontal 
{
   background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(150, 150, 150, 255));
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/scrollgriphorizontal.png&quot;);
   min-width: 3em;
   border-color: rgb(40, 40, 40);
   border-style: solid;
   border-radius: 3px;
}

QScrollBar::add-line:horizontal 
{
   border-color: rgba(40, 40, 40, 0);
   width: 18px;
   subcontrol-position: right;
   subcontrol-origin: margin;
}

QScrollBar::sub-line:horizontal 
{
   border-color: rgba(40, 40, 40, 0);
   width: 18px;
   subcontrol-position: left;
   subcontrol-origin: margin;
}

QScrollBar::left-arrow:horizontal
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/leftarrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::left-arrow:horizontal:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::right-arrow:horizontal 
{
   width: 17px;
   height: 17px;
   image: url(&quot;:/resources/rightarrow.png&quot;);
   background-color: qlineargradient(spread:pad, x1:0.767, y1:1, x2:1, y2:0, stop:0 rgba(110, 110, 125, 255), stop:1 rgba(150, 150, 160, 255));
   border-radius: 2px;
}

QScrollBar::right-arrow:horizontal:pressed
{
  color: rgb(190, 190, 190);
  background-color: rgba(50, 50, 50, 255);
  border-style: inset;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal 
{
   background: none;
}

QGroupBox
{
   background-color: rgba(120, 120, 120, 255);
   border-radius: 5px;
   font: 12px;
   color: rgb(255,255,255);
   margin-top: 3ex;    
}

QGroupBox::title
{
   subcontrol-origin: margin;
   padding: 0 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="elementGroupBox">
     <property name="title">
      <string>Element Name:</string>
     </property>
     <layout class="QHBoxLayout" name="elementLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="whatsThis">
         <string>Enter the new element's name here.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="helpToolButton">
        <property name="whatsThis">
         <string>Display a detailed &quot;How To&quot;.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/gcresources.qrc">
          <normaloff>:/resources/helpcursor.png</normaloff>:/resources/helpcursor.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="attributesGroupBox">
     <property name="title">
      <string>Attribute Names:</string>
     </property>
     <layout class="QVBoxLayout" name="attributesLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="whatsThis">
         <string>Enter the associated attribute names here.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="whatsThis">
        <string>Add the new element and all (if any) attributes.</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="donePushButton">
       <property name="whatsThis">
        <string>Close this form when you're finished adding elements.</string>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/gcresources.qrc"/>
 </resources>
 <connections/>
</ui>
